src/SignalR/server/StackExchangeRedis/test/Docker.cs

```diff
@@ -81,6 +81,9 @@ private void StartRedis(ILogger logger)
             {
                 logger.LogError(ex, "Error starting redis docker container, retrying.");
                 Thread.Sleep(1000);
+
+                // Call stop just in case the container somehow started after the timeout so our retry logic doesn't fail
+                RunProcessAndWait(_path, $"stop {_dockerContainerName}", "docker stop", logger, TimeSpan.FromSeconds(15), out var _);
```

Comment:
Will this throw if the container is already stopped?

Reply:
Nope, that would be `RunProcessAndThrowIfFailed`